addons/hr/models/hr_employee_base.py

```diff
@@ -74,6 +74,7 @@ class HrEmployeeBase(models.AbstractModel):
     show_hr_icon_display = fields.Boolean(compute='_compute_presence_icon')
     im_status = fields.Char(related="user_id.im_status")
     newly_hired = fields.Boolean('Newly Hired', compute='_compute_newly_hired', search='_search_newly_hired')
+    has_address_work_phone = fields.Boolean(default=True)
```

Comment:
Hi !

This is overkill, this will probably won't work for existing customers as no migration strategy is defined, and I fear this will be source of desynchronization. On the other hand, we try to avoid stored field for that kind of usage.

Reply:
@ataha1 Yes I checked with the product owner, and he said that it would be better (because less magical at the end). Sorry I forgot to keep you updated about the spec change.

Thanks for your work ! :)